pep_sphinx_extensions/pep_zero_generator/subindicies.py

```diff
@@ -0,0 +1,50 @@
+"""Utilities to support sub-indices for PEPs."""
```

Comment:
The filename is misspelled (subindicies -> subindices)

Reply:
Perils of typing past midnight! Thanks for noticing.

A

Reply:
Also in function/variable names in this file.

And it would be clearer to use the same naming as the display URL, so `topics` instead of `subindices`

Reply:
> And it would be clearer to use the same naming as the display URL, so topics instead of subindices

The one issue with that, though, is that it can quite easily be extended to generate subindicies filtered on other headers than `Topic`, e.g. `Type`, `Status`, a future such field or potentially something else; the code isn't topic-specific.